flow/project.py

```diff
@@ -2980,10 +3197,9 @@ def submit(self, bundle_size=1, jobs=None, names=None, num=None, parallel=False,
         :type ignore_conditions:
             :py:class:`~.IgnoreConditions`
         """
-        # Regular argument checks and expansion
-        if jobs is None:
-            jobs = self  # select all jobs
+        aggregates = self._convert_aggregates_from_jobs(jobs)
```

Comment:
I just wanted to note that here we validation the aggregates since it is in our public API, but we have already done this in `_select_jobs_from_args` on the command line side. Changing this is out of scope. I just wanted to document it.

Reply:
Move to #362.